### ISSUE TITLE
Make controllers more boring (needs to be more RESTful)

### ISSUE THREAD

**Issue by thebrianemory** [thebrianemory/schedule-app#2]
There are a good amount of areas in my controllers that are not RESTful. For example, from the schedules_controller:
```
def add_employees
  Employee.admins_employees(current_admin).each do |employee|
    @schedule.employees << employee if employee.status == 'active'
  end

  redirect_to schedule_path(@schedule)
end

def remove_employees
  @schedule.employees.delete_all

  redirect_to schedule_path(@schedule)
end

def set_schedule
  @set_schedule = true
end

def print_schedule
  @printable_view = true
end
```

I need to abstract a lot of things into separate controllers, keeping it CRUDy.

**Comment by thebrianemory** [comment]
Commit https://github.com/thebrianemory/schedule-app/commit/9508ddfd03cad70281f0eed8670182de2ff9fd0f is the first of many to improve the RESTfulness of the app.

**Make the creation of shifts more RESTful** 
This commit is across several files which is not ideal. I chose to do so
to keep it all in one commit due to the extent of the changes that I was
making. This commit moves the creation of shifts from the schedules
controller to the shifts controller. The original implementation worked
but was not at all RESTful.

In the schedules_controller, I removed the set_schedule action and moved
it to the index action in the shifts_controller. This action is acting
more like the new and edit actions rolled into one. When an admin clicks
the 'Set schedule' button, it hits the index action regardless if this
is to add shifts for the first time or to update created shifts. I can
take the pursuit of REST a step further by adjusting the button to hit
either the new or edit action depending on if the schedule already has
any created shifts or not.

The rest of the file changes were made to get the tests passing that
were broken when I made the above change.
